jscomp/bsb/bsb_clean.ml

```diff
@@ -49,6 +49,8 @@ let clean_bs_garbage proj_dir =
     Bsb_parse_sources.clean_re_js proj_dir; (* clean re.js files*)
     ninja_clean  proj_dir ;
     Ext_list.iter Bsb_config.all_lib_artifacts try_remove ;
+    try_remove (Bsb_config.lib_lit // (Filename.basename !Bsb_global_backend.lib_ocaml_dir));
+    try_remove (Bsb_config.lib_lit // (Filename.basename !Bsb_global_backend.lib_artifacts_dir));
```

Comment:
I suppose this is already done by `ninja_clean`?

Reply:
I think ninja wouldn't know if you've deleted one of your source files and tried to clean the artifacts right? This was a way for us to ensure everything was cleaned up.

Reply:
this is already done here : `Ext_list.iter Bsb_config.all_lib_artifacts try_remove`

Reply:
So ` all _lib_artifacts` uses a static array of possible artifacts directory, which won’t contain the new artifacts directory that we introduced with the support of `-backend`: which are `bs-native`, `ocaml-native` and `bs-bytecode`, `ocaml-bytecode`. I didn’t want to add them to the array because then we would be cleaning all artifacts (js, bytecode and native) instead of the one specified by the `-backend` field. But I could be convinced that it’d be better to just clean everything